Welcome.md

```diff
@@ -44,6 +47,33 @@ start using or contributing to our code.
     - [Prevent wrapping within psql (optional)](#prevent-wrapping-within-psql-optional)
 
 
+## Open source philosophy
+The Seattle Flu Study practices open science.
+This means that our code (with very few, notable exceptions) is all open source.
```

Comment:
suggestion to include documentation too:
This means that our code and documentation (with very few, notable exceptions) are all open source.